packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue

```diff
@@ -0,0 +1,186 @@
+    RplAnchorLinks,
+    RplDivider,
+    RplRow,
+    RplCol
+  },
+  props: {
+    sidebar: Boolean
+  },
+  head () {
+    return {
+      title: this.publication.title
+    }
+  },
+  methods: {
+    formatAnchor (title) {
+      return kebabCase(title)
+    }
+  },
+  mounted () {
+    const printWindow = () => {
+      window.print()
+    }
+    this.$nextTick(function () {
+      if (process.client && typeof window !== 'undefined') {
+        setTimeout(printWindow(), 3000)
+      }
+    })
+  },
+  computed: {
+    breadcrumbs () {
+      return [{ text: 'Home', url: '/' }, { text: this.publication.title, url: `/${this.$route.params.publicationname}` }, { text: 'Print', url: this.$route.path }]
+    },
+    anchorLinks () {
+      return this.pages.map(page => ({
+        url: `#${this.formatAnchor(page.title)}`,
+        text: page.title
+      }))
+    },
+    publishingInfo () {
+      if (this.publication.type === 'node--publication') {
+        const author = this.publication.field_publication_authors && this.publication.field_publication_authors.length > 0 ? this.publication.field_publication_authors.join(', ') : ''
+        const date = this.publication.field_publication_date
+        const copyright = this.publication.field_license_type ? this.publication.field_license_type.description : ''
+        return (author || date || copyright) ? { author, date, copyright } : null
+      }
+    },
+    pages () {
+      return this.publicationPages.map(page => {
+        return {
+          title: page.title,
+          components: page.componentMapping.map(cmp => {
+            return this.$tide.getDynamicComponent(cmp, true)
+          }).filter(c => c)
+        }
+      })
+    }
+  },
+  async asyncData (context) {
+    const publication = await context.app.$tide.getPageByPath('/' + context.route.params.publicationname)
+    const hierarchyJson = await context.app.$tide.get('node/publication', {}, `${publication.id}/hierarchy`)
+    const ids = []
+    const addId = (o, arr) => {
+      if (o.id) {
+        arr.push(o.id)
+      }
+      if (o.children) {
+        o.children.forEach(gc => {
+          addId(gc, ids)
+        })
+      }
+    }
+
+    addId(hierarchyJson.data.meta.hierarchy, ids)
+
+    const filters = {
+      pub: {
+        condition: {
+          operator: 'IN',
+          path: 'id',
+          value: {
+            '': ids
+          }
+        }
+      }
+    }
+
+    const includeConfig = require('./../tide.config')
+    const includes = includeConfig.include.publicationPage
+
+    const publicationPagesData = await context.app.$tide.getContentList('publication_page', filters, includes)
+    const publicationPages = []
+    async function addComponents () {
+      for (let i = 0; i < publicationPagesData.length; i++) {
+        const page = publicationPagesData[i]
+        const componentMapping = await context.app.$tideMapping.getMapping(page.field_landing_page_component, 'landingPageComponents')
```

Comment:
`$tideMapping.get()` should work, https://github.com/dpc-sdp/ripple/blob/develop/packages/ripple-nuxt-tide/lib/core/middleware.js#L133

Reply:
it doesn't though, its not exposed in the constructor (try consoling out this.$tideMapping)

Reply:
@dylankelly as discussed, please remove the get in constructor.